### PR TITLE
Optimize-routing

### PR DESCRIPTION
**Pull request by HexaField**
## References
closes #_insert number here_

### CODE REVIEW

**Comment by HexaField** [comment]
Worth deploying this to QAT and ensuring auth login still works

File: packages/client-core/src/admin/components/settings/tabs/client.tsx [hunk 1]
```diff
@@ -130,24 +104,34 @@ const ClientTab = forwardRef(({ open }: { open: boolean }, ref: React.MutableRef
...
         state.set({ loading: false, errorMessage: e.message })
       })
   }
 
   const handleCancel = () => {
-    settings.set(clientSetting)
+    settingsState.set(clientSettings)
   }
 
+  console.log(settingsState.value)
```
Comment (aditya-mitra) [inline packages/client-core/src/admin/components/settings/tabs/client.tsx]: we can remove this log statement

**Review by aditya-mitra — APPROVED** [review]
Looks good!

File: packages/client-core/src/admin/components/settings/tabs/client.tsx (hunk 1 above)
Comment (HexaField) [reply packages/client-core/src/admin/components/settings/tabs/client.tsx]: done

**Pull request merged**